src/excel.js

```diff
@@ -12,28 +12,24 @@
 
 'use strict';
 
-const { ServiceBusClient } = require('@azure/service-bus');
+const { SQSClient, SendMessageCommand } = require('@aws-sdk/client-sqs');
+const uuid = require('uuid');
 
 const mapResult = require('./mapResult.js');
 
-/**
- * Excel provider that assembles all excel based indices to optimize the number of
- * messages sent to the queue. It collects all the index update and delete
- * operations for a path and packs them into one message sent to the queue.
- */
 class Excel {
   constructor(params, configs, log) {
     const {
-      owner, repo, ref,
+      owner, repo,
+      AWS_REGION: region,
+      AWS_ACCOUNT_ID: accountId,
+      AWS_SQS_QUEUE_NAME: queueName = `https://sqs.${region}.amazonaws.com/${accountId}/helix-excel-${owner}-${repo}.fifo`,
```

Comment:
usually we have double dashes, eg:

```suggestion
      AWS_SQS_QUEUE_NAME: queueName = `https://sqs.${region}.amazonaws.com/${accountId}/helix-excel--${owner}--${repo}.fifo`,
```